modules/core/src/main/scala/CharCollector.scala

```diff
@@ -51,14 +51,19 @@ private object CharCollector:
     curState match
       case State.Init =>
         char match
+          case 3|4 => // Ctrl+C or Ctrl+D
```

Comment:
I think it's worth gating this case to only run on windows (as with the others that you added for windows support)

Reply:
Updated in [74fd4ea](https://github.com/neandertech/cue4s/pull/36/commits/74fd4ea7d13ae5ff832fdcc4997eccdae170e5df) and [ce1f252](https://github.com/neandertech/cue4s/pull/36/commits/ce1f2527cd5b5cc0432f7e2708337e6dd3bb131b)